client/state/selectors/get-inline-help-support-variation.js

```diff
@@ -31,7 +32,7 @@ export default function getSupportVariation( state ) {
 		return SUPPORT_TICKET;
 	}
 
-	if ( getCurrentUserLocale( state ) === 'en' && isDirectlyReady( state ) ) {
+	if ( startsWith( getCurrentUserLocale( state ), 'en' ) && isDirectlyReady( state ) ) {
```

Comment:
Looks like we have English locales in config: https://github.com/Automattic/wp-calypso/blob/26ae6d6538be9ffada428c3b2017c8336d0c6b36/config/_shared.json#L146 and in the i18n package: https://github.com/Automattic/wp-calypso/blob/e801aaa443dfe1c1d81a5315efbd1f35682a5f61/packages/i18n-utils/src/locales.ts#L25
Might be a good idea to use them instead - just to make it easier to find where we're checking for English locales.

Reply:
Sure, I was trying to use the trick surfaced from pxLjZ-6AY-p2:

> the language check should compare the first two characters of the user locale to `en` instead of doing a strict `'en' === $locale` check

But I'm seeing also that you're suggesting a utility function to handle that. So it's probably better to stick with one of the existing methods, which can be migrated together.